app/connectors/ReferenceDataConnector.scala

```diff
@@ -60,6 +60,11 @@ class ReferenceDataConnector @Inject() (config: FrontendAppConfig, http: HttpCli
     http.GET[Seq[AdditionalInformation]](serviceUrl, headers = version2Header)
   }
 
+  def getMethodOfPaymentTypes()(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[Seq[MethodOfPayment]] = {
+    val serviceUrl = s"${config.referenceDataUrl}/lists/MethodOfPayment"
```

Comment:
Is this the correct list name? I thought it was `TransportChargesMethodOfPayment`?

Reply:
This will affect your customs-reference-data branch as well ^

Reply:
we put just MethodOfPayment, as we may want to use it in other pages as well..

Reply:
Needs to match the code list name from Europa

Reply:
In production, we ingest the data and save it in mongo with that list name. If that list name doesn't match then it will fail to fetch the data.

Reply:
ok will change it, thanks Jack for pointing out this one.